big-practice/src/style/components/modal-table-add.css

```diff
@@ -0,0 +1,87 @@
+/* START MODAL */
```

Comment:
Model file should be for modal only, not for specific function like table, form or etc